mrbelvedereci/cumulusci/views.py

```diff
@@ -24,6 +28,36 @@ def org_detail(request, org_id):
     } 
     return render(request, 'cumulusci/org_detail.html', context=context)
     
+def org_lock_unlock(request, org_id, action):
```

Comment:
it took me a minute to realize this wasn't actually wired up directly to a urlconf - can you add a comment or do we have a naming convention we can use to make that quicker to notice? just a thought.

Reply:
Sure, I'll have it start with underscore and add a comment to make it clear.